### PR TITLE
Add documentation on log level

### PR DESCRIPTION
**Pull request by eliecharra**
| Q                 | A
| ----------------- | ---
| 🐛 Bug fix?       | no
| 🚀 New feature?   | no
| ⚠ Deprecations?   | no
| ❌ BC Break       | no
| 🔗 Related issues | #256 
| ❓ Documentation  | yes

## Description

Please include a summary of the change and which issue is fixed.
Please also include relevant motivation and context.

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -12,6 +12,24 @@ When a crash occurs in driftctl, we do not send any crash reports.
 For debugging purposes, you can add `--error-reporting` when running driftctl and crash data will be sent to us via [Sentry](https://sentry.io)
 Details of reported data can be found [here](./cmd/flags/error-reporting.md)
 
+#### Log level
+
+By default driftctl logger only display warning and error messages. You can set `LOG_LEVEL` environment variable to change the default level.
```
Comment (wbeuil) [inline doc/README.md:17]: `only displays ...`

File: doc/README.md [hunk 2]
```diff
@@ -12,6 +12,24 @@ When a crash occurs in driftctl, we do not send any crash reports.
...
+Valid values are : trace,debug,info,warn,error,fatal,panic.
+
+**Note:** In trace level, terraform provider logs will be shown.
+
+Example
+
+```shell
+LOG_LEVEL=debug driftctl scan
```
Comment (wbeuil) [inline doc/README.md:25]: `$ LOG_LEVEL=debug driftctl scan`

File: doc/README.md [hunk 3]
```diff
@@ -24,4 +42,4 @@ Details of reported data can be found [here](./cmd/flags/error-reporting.md)
 
 ## Issues
 
-- [Known Issues & Limitations](LIMITATIONS.md)
\ No newline at end of file
+- [Known Issues & Limitations](LIMITATIONS.md)
```
Comment (wbeuil) [inline doc/README.md:45]: I'm blind I don't see the diff here ...

**Review by wbeuil — CHANGES_REQUESTED** [review]

File: doc/README.md (hunk 3 above)
Comment (eliecharra) [reply doc/README.md:45]: Only a newline missing at the end of the file added automatically sorry for the noise ...

**Review by wbeuil — APPROVED** [review]

**Pull request merged**